be/src/vec/data_types/serde/data_type_number_serde.h

```diff
@@ -210,7 +210,7 @@ void DataTypeNumberSerDe<T>::write_one_cell_to_jsonb(const IColumn& column,
                                                      int row_num) const {
     result.writeKey(col_id);
     StringRef data_ref = column.get_data_at(row_num);
-    if constexpr (std::is_same_v<T, Int8>) {
+    if constexpr (std::is_same_v<T, Int8> || std::is_same_v<T, UInt8>) {
         int8_t val = *reinterpret_cast<const int8_t*>(data_ref.data);
```

Comment:
if the value is uint8, but cast it to int8_t at line 214, maybe something lost?

Reply:
doris has no uint8 type, uint8 is for boolean, so lost is not possible here